### DISCUSSION TITLE
Auto dropdown placement

### DISCUSSION THREAD

**Discussion by kjvdven** [orchidjs/tom-select#283] [Ideas]
Nice plugin! 

I have a Feature request.

When the select is at the bottom of a page, I suspect the dropdown to be positioned above  the input automatically, like select2 does.

**Tom select**
![image](https://user-images.githubusercontent.com/306806/143675724-431e3d1c-c0b5-4652-aa53-1df0d905f01b.png)

**Select2**
![image](https://user-images.githubusercontent.com/306806/143675786-75c23378-729f-413c-9b0b-d42406f29d9d.png)

**Comment by oyejorge** [comment]
See #163
